lib/actions/apiV2.js

```diff
@@ -1224,6 +1225,14 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
                     })
                   }
                 }
+                if (
+                  displayMicromobilityRentalError &&
+                  query.variables.modeButtons.includes('rent')
```

Comment:
Should we display this anytime the user selected rent or only if the response includes any rental itineraries?